integration/coverage_imports.go

```diff
@@ -0,0 +1,78 @@
+// +build integration
```

Comment:
This is generated, you may want to delete it (we should probably add to the .gitignore) cc @prateek

Reply:
Yeah sorry, I think it might have gotten pulled in after `git add .` after the merge